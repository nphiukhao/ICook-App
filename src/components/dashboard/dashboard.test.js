import React from 'react'
import ReactDOM from 'react-dom';
import Dashboard from './dashboard'
import { MemoryRouter } from 'react-router-dom'


it(`Renders without crashing`, () => {
    const div = document.createElement('div')
    ReactDOM.render(<MemoryRouter><Dashboard /></MemoryRouter>, div)
    ReactDOM.unmountComponentAtNode(div);
});