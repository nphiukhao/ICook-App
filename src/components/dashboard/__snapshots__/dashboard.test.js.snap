// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component Dashboard snapshot test 1`] = `
<div
  className="dashboard-container"
>
  <div
    className="landing-section"
  >
    <p
      className="dashhead"
    >
      don't know what to cook? 
      <span>
        use ICook
      </span>
    </p>
    <p
      className="describution"
    >
      ICook is a recipe application built to save time, money, and headaches. Input your all your favorite recipes and let this application do all the work!
    </p>
    <a
      className="start"
      href="#start"
      style={
        Object {
          "textDecoration": "none",
        }
      }
    >
      explore now
    </a>
  </div>
  <section
    className="get-started"
    id="start"
  >
    <p
      className="welcome"
    >
      Welcome to ICook!
    </p>
    <div
      className="options"
    >
      <Link
        className="dashboard-div"
        style={
          Object {
            "textDecoration": "none",
          }
        }
        to="/all"
      >
        <p>
           See All Recipes
        </p>
        <div
          className="allfiltLogo"
        />
      </Link>
      <Link
        className="dashboard-div"
        style={
          Object {
            "textDecoration": "none",
          }
        }
        to="/time"
      >
        <p>
           Filter by Time 
        </p>
        <div
          className="timefiltLogo"
        />
      </Link>
      <Link
        className="dashboard-div"
        style={
          Object {
            "textDecoration": "none",
          }
        }
        to="/ingredient"
      >
        <p>
           Filter by Ingredients 
        </p>
        <div
          className="infiltLogo"
        />
      </Link>
    </div>
  </section>
</div>
`;
